benchmarks/data-caching/client/Dockerfile

```diff
@@ -8,12 +8,16 @@ ENV DEBIAN_FRONTEND noninteractive
 
 COPY memcached.tar.gz /memcached.tar.gz
 
-RUN apt-get update && apt-get install -y libevent-2.1-6 vim --no-install-recommends \
-	&& rm -rf /var/lib/apt/lists/*
-
-RUN buildDeps='curl gcc libc6-dev libevent-dev make' \
+RUN buildDeps='curl gcc libc6-dev libevent-dev make wget ca-certificates' \
```

Comment:
We should have `vim` (or another editor) installed because the user may need to change the file in which the server addresses are written.

Reply:
Updated